.github/workflows/integrate.yaml

```
@@ -17,32 +17,42 @@ jobs:
 
   lint:
     name: Lint Check
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
 
     steps:
       - name: Check out code
         uses: actions/checkout@v3
 
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v5.3.0
+        with:
+          python-version: 3.8
+
       - name: Install dependencies
-        run: sudo apt-get install python3-pip tox
+        run: python3 -m pip install tox
```

nitpick: Equivalent, but for consistency with our other repos
```suggestion
        run: pip install tox
```